### ISSUE TITLE
Media Player Elements

### ISSUE THREAD

**Issue by SoNotMC** [aleksilassila/reiverr#145]
I would like more elements to be in the media player. Such as a volume slider/mute button, end time, liking an episode/movie, playback settings, picture in picture, all the relative play/pause + skip + jump fw and bw and the star rating of episodes/movies.

Just feel like it is way to simple right now and lacking lots of needed features.

**Comment by aleksilassila** [comment]
Agreed, I'll address this when I retouch the video player and or add sections

**Comment by SoNotMC** [comment]
Sounds good, thank you.
